charts/capsule/Chart.yaml

```diff
@@ -21,8 +21,15 @@ sources:
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.9
+version: 0.2.0
```

Comment:
Please, don't bump up the chart since we're going to address it before the Capsule release!

Reply:
Reverted Chart Bump to 0.1.9